src/azure-cli/requirements.py3.Linux.txt

```diff
@@ -31,7 +31,7 @@ azure-mgmt-compute==11.0.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==1.5.0
 azure-mgmt-containerregistry==3.0.0rc9
-azure-mgmt-containerservice==8.1.0
+azure-mgmt-containerservice==9.0.0
```

Comment:
major version bump requires a full live test regression of whole commands

Reply:
Manually tested all commands on my test cluster

Reply:
thanks xiazhmsft. @qwordy  this is another case where live test is really useful. let's nail it down in this sprint, thanks.